code/datums/martial/cqc.dm

```diff
@@ -97,7 +97,7 @@
 		D.visible_message("<span class='warning'>[A] locks [D] into a restraining position!</span>", \
 							"<span class='userdanger'>[A] locks you into a restraining position!</span>")
 		D.apply_damage(damage, STAMINA)
-		D.Stun(100)
+		D.Stun(10)
```

Comment:
I think the reason this exists is because otherwise, CQC is still at the mercy of RNG grabs. Of course, the core issue with Restrain is that it is a combo, and because it is a combo, even if the initial grab fails and they break free, the second grab, which completes the restrain combo, will always succeed and stun.

This isn't necessarily a problem with CQC specifically, but actually a problem with grabs being extremely all-or-nothing.

Reply:
admittedly yeah, it still exists because rng grabs are cruel and usually will just end up irritate whoever's using the martial art, this gives it a little bit more of a base to work with while still giving someone the chance to escape shortly after. combo-stunning is goofy but i think this should make it far better at fighting single targets, rather than being able to chain-stun multiple people in quick succession, and keep them there for 10s per person.